templates/documentation.html

```diff
@@ -69,16 +69,6 @@ <h3 class="text-center">
                   </a>
                 </div>
               </div>
-              <div class="list-group">
-                <a class="list-group-item" href="https://inveniordm.docs.cern.ch/deployment/">
-                  <i class="fa fa-fw fa-rocket"></i>
-                  Deploy
```

Comment:
There is a [WIP](https://github.com/inveniosoftware/docs-invenio-rdm/pull/435) pr for this.

Reply:
Should we wait for this to be merged then?

Reply:
you can remove this!